ang/civicase/CaseCard--case-list.html

```diff
@@ -4,15 +4,15 @@
   * use 'civicase__animated-checkbox-card--expanded' class : for enabled checkbox state
```

Comment:
Do we still need this?

Reply:
No we don't need this. Let me remove it.